hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEngine.java

```diff
@@ -156,16 +157,18 @@ public HapiTestEngineExecutionContext before(HapiTestEngineExecutionContext cont
                 return context;
             }
 
-            env = new HapiTestEnv("HAPI Tests", true);
-            env.start();
+            // For now, switching to non-in process servers, because in process doesn't work for the
+            // restart and reconnect testing.
+            env = new HapiTestEnv("HAPI Tests", true, false);
```

Comment:
Why the second argument is always **true**? I also saw that you removed the cluster variable from the HapiTestSuite annotation. How do we plan to control that?

Reply:
Hmmm. We weren't actually making use of the "cluster" flag anywhere. But we could. We could default to false and then in special cases (where we expect to take a node down) set `cluster` to true. But I thought this was maybe less useful, for two reasons. First, there is value in running a cluster all the time, because it helps to find ISS issues. Second, if one node happens to die for some random non-code reason, the network keeps running.

On the other hand, if a node dies for some code reason, then we won't catch it.

It would be almost trivial to support the flag and have "cluster" default to false.